src/helpers/__tests__/__snapshots__/helpers.test.ts.snap

```diff
@@ -5,6 +5,9 @@ exports[`getAuthType should return a credential type: credentialTypes 1`] = `
   "Username and Password",
   "Token",
   "SSH Key",
+  "SSH Key File",
+  "Unknown credential type",
+  "Unknown credential type",
```

Comment:
Is this duplicated "Unknow cred..." entry necessary?

Reply:
on the test we see 

```js
      generateAuthType({ auth_type: 'lorem' }),
      generateAuthType({ auth_type: 'ipsum' })
```

so yeah, that's the expected response